.github/workflows/buid_and_release.yml

```diff
@@ -16,5 +16,20 @@ jobs:
     - uses: actions/checkout@v2
     - name: build
       run: make all
+    - name: Get current time
+      uses: josStorer/get-current-time@v2.0.2
+      id: current-time
+      with:
+        format: YYYY-MM-DD
     - name: Release
-      run: make deploy-release
+      run: echo Uploading files as new release.
+    - uses: "marvinpinto/action-automatic-releases@latest"
+      with:
+        repo_token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
@matentzn To be honest, I'm really not sure how this authorization works. It has been working for OMIM, so I re-used that and it's working here too. But there is no repository secret set up called `GITHUB_TOKEN`. And I don't have a personal access token / secret on my account called that either. It looks like I just stuck with the default here: https://github.com/marvinpinto/action-automatic-releases, but it gives no explanation about it. So I'm kind of worried about this possibly breaking in the future.

_edit: never mind, figured it out. This is just a feature of GitHub and a token that is invisible to settings but automatically set up: https://github.blog/changelog/2021-04-20-github-actions-control-permissions-for-github_token_